api/envoy/api/v2/core/base.proto

```diff
@@ -188,8 +188,7 @@ message HeaderValueOption {
   google.protobuf.BoolValue append = 2;
 
   // Whether the header should be modified when it is already present in the request or response.
-  // Defaults to false.
-  google.protobuf.BoolValue skip_if_present = 3;
+  bool skip_if_present = 3;
```

Comment:
This is a breaking change, so I don't think we can change it. This is more an "in principle" argument, since Envoy wan't using it before, but we hadn't marked it as alpha or not-implemented. I'm willing to bet that no other client or management server is setting it either as a result, but I might be wrong. @mattklein123 @lizan second opinion?

Reply:
Sorry what is the breaking change? I think you are looking at an intermediate diff?

Reply:
Yes, my bad.